pkg/controller/jobframework/integrationmanager.go

```diff
@@ -209,3 +211,25 @@ func GetEmptyOwnerObject(owner *metav1.OwnerReference) client.Object {
 	}
 	return nil
 }
+
+// GetMultiKueueAdapters returns the map containing the MultiKueue adaptors of the
+// registered integration.
+// An error is returned if more then one adapter is registers for one object type.
+func GetMultiKueueAdapters() (map[string]MultiKueueAdapter, error) {
+	ret := map[string]MultiKueueAdapter{}
+	err := manager.forEach(func(_ string, cb IntegrationCallbacks) error {
```

Comment:
I would suggest to inline the check for `err != nil`

Reply:
done